ui/routes.py

```diff
@@ -305,6 +305,15 @@ def on_get(self, _: falcon.Request, resp: falcon.Response, worker_doc: str):
             raise falcon.HTTPNotFound(title="Docs not found", description="'{}' is not available".format(worker_doc))
 
 
+class BackendDetails:
```

Comment:
The name of the class does not adhere to the general naming convention. Please rename this into BackendDetailsView

Reply:
Ok - I have also modified ``WorkerDocs`` to match this convention